katsdpsigproc/tune.py

```diff
@@ -109,7 +111,9 @@ def _db_keys(fn: _TuningFunc, args: Sequence, kwargs: Mapping) -> Dict[str, Any]
     return keys
 
 
-def _assemble_query(tablename: str, keys: Mapping[str, Any]) -> Tuple[str, List]:
+def _assemble_query(
```

Comment:
Maybe rename it now that it actually runs the query instead of just assembling it?